app/_src/mesh/sbom.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Software Bill of Materials
+toc: false
+---
+
+A software bill of materials (SBOM) is an inventory of all software components (proprietary and open source), open source licenses, and dependencies in a given product. A software bill of materials (SBOM) provides visibility into the software supply chain and any license compliance, security, and quality risks that may exist.
+
+Starting in {{site.mesh_product_name}} 2.7.4, we are generating SBOMs for {{site.mesh_product_name}} and docker container images.
+
+1. [Download security assets](https://packages.konghq.com/public/kong-mesh-binaries-release/raw/names/security-assets/versions/{{page.version}}/security-assets.tar.gz) for the latest version of {{site.mesh_product_name}}
```

Comment:
The `{{page.version}}` should point to the latest minor version of the page for any specific `major.minor.x`release branch / page. 

Ex: For instance, if `2.7.5 for 2.7.x`, `2.8.1 for 2.8.x`, `2.10.0 for 2.10.x` are released, then each of the pages will point to these latest versions. However, if someone needs SBOM for `2.8.0` instead of `2.8.1` then they will have to be shared with the customer from cloudsmith.  

@lahabana Can you clarify if this is what you wanted / required for docs to always point to latest minor patch cloudsmith links ?

Reply:
thanks for the clarification.